surveys/urls.py

```diff
@@ -8,51 +8,73 @@
 from django.conf.urls import url, include
 from django.utils.translation import ugettext_lazy as _
 from django.contrib.auth.decorators import login_required
-from surveys.views import AgencyCreate, LocationCreate, StudyCreate, \
-                            StudyAreaCreate, SurveyCreate, SurveyPublish, \
-                            SurveyListEdit, SurveyListRun, Signup, SurveySubmittedList, \
-                            SurveySubmittedDetail, SurveyPropertiesEdit
+from surveys import views as survey_views
+
 import fobi.views
 
 urlpatterns = [
     # placeholder view for a public-facing landing page
     url(r'^$',
-        login_required(SurveyListRun.as_view()),
+        login_required(survey_views.SurveyListRun.as_view()),
         name='home'),
 
     url(r'^accounts/',
         include('django.contrib.auth.urls')),
 
     url(r'^accounts/signup$',
-        Signup.as_view(),
+        survey_views.Signup.as_view(),
         name='signup'),
 
     url(r'agencies/create/$',
-        login_required(AgencyCreate.as_view()),
+        login_required(survey_views.AgencyCreate.as_view()),
         name='agencies-create'),
 
     url(r'locations/create/$',
-        login_required(LocationCreate.as_view()),
+        login_required(survey_views.LocationCreate.as_view()),
         name='locations-create'),
 
+    url(r'locations/list/$',
```

Comment:
This is more of a style comment, but I wanted to call out here that our URLpatterns have started to diverge from the [REST guidelines](https://medium.com/hashmapinc/rest-good-practices-for-api-design-881439796dc9) that we were working with earlier. In particular:

> 3. Let the HTTP Verb Define Action

Following this advice, I think `/entities/list/` and `/entities/detail/<pk>/` put too much of the verb in the URL, and I wonder if they could be replaced by `/entities/` and `/entities/<pk>/` respectively. (In this case, you could swap `locations` or `studies` for `entities`.)

There's some amount of necessary divergence we need to take from this guideline, e.g. since we're using a `DeleteView` for `delete` actions that accepts both `GET` and `POST` requests it makes a lot of sense to have `/entities/delete/` routes. But for `list` and `detail` routes I wonder whether we could get away with simpler URLs.

Reply:
I like it!